lib/manager/azure-pipelines/index.ts

```diff
@@ -0,0 +1,5 @@
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  fileMatch: ['(^|/)azure-pipelines?\\.ya?ml'],
```

Comment:
This is a tricky one, as you can select any `.yaml` or `.yml` file when setting up a pipeline, however you're correct that these filenames are the convention.

I would, however, add some more. I've also seen `.azure-pipelines.yaml` (and all the variations) as well as a directory called `.azure-pipelines` and any `.yaml`/`.yml` file under that directory.